fili-core/src/main/java/com/yahoo/bard/webservice/web/DefaultResponseFormatResolver.java

```diff
@@ -0,0 +1,42 @@
+// Copyright 2017 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.web;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import javax.ws.rs.container.ContainerRequestContext;
+
+/**
+ * A Fili default implementation of ResponseFormatResolver. This implementation works with three formats: json, jsonapi
+ * and csv.
+ */
+public class DefaultResponseFormatResolver implements ResponseFormatResolver {
+    public static final String ACCEPT_HEADER_JSON = "application/json";
+    public static final String ACCEPT_HEADER_JSONAPI = "application/vnd.api+json";
+    public static final String ACCEPT_HEADER_CSV = "text/csv";
+    private final Map<String, String> formatsMap;
+
+    /**
+     * Constructor.
+     */
+    public DefaultResponseFormatResolver() {
+        formatsMap = new LinkedHashMap<>();
+        formatsMap.put(ACCEPT_HEADER_JSON, "json");
+        formatsMap.put(ACCEPT_HEADER_JSONAPI, "jsonapi");
+        formatsMap.put(ACCEPT_HEADER_CSV, "csv");
+    }
```

Comment:
"it's probably a good idea to pull the format names for json, jsonapi and csv out into public constant strings. They could live in this class, but I could make the case for putting them in other classes as well."

I was actually more concerned about the values themselves.  They should be refactored with constants here, in the response builder and in the tests.